### PR TITLE
feat(docs): Rebranding the entire Documentation

### PR DESCRIPTION
**Pull request by riccardopersiani**
Rebranding Oraclize to Provable.

Please @D-Nice, review. 
Changed from Oraclize to Provable just where is not code.

### CODE REVIEW

File: source/includes/_corda.md [hunk 1]
```diff
@@ -116,13 +116,13 @@ override fun verify(tx: LedgerTransaction) {
...
         ...
 
-        // Check tha Oraclise's answer 
+        // Check tha Oraclise's answer
         val rate = answerCommand.value.rawValue as String
         "The rate USD/GBP must be over $USD_GBP_RATE_THRESH" using (rate.toDouble() > USD_GBP_RATE_THRESH)
 
         // You can use the proof verification tool in the contract as well
         val proofVerificationTool = OraclizeUtils.ProofVerificationTool()
-        "Oraclize's proof verification failed" using  (
```
Comment (riccardopersiani) [inline source/includes/_corda.md:125]: @D-Nice not sure if this is worth the change

Comment (D-Nice) [reply source/includes/_corda.md:125]: yep dont worry about yet, that will be changed as well, once i have our code rebrand complete as well

**Review by D-Nice — APPROVED** [review]
LGTM

but would be worthwhile for you to inspect my latest commit

**Comment by D-Nice** [comment]
please merge and deploy

**Comment by riccardopersiani** [comment]
@D-Nice nice catches with that commit.

**Pull request merged**